plugins/usepe_logger.py

```diff
@@ -31,14 +37,16 @@ def init_plugin():
     # Create the loggers
     global conflog
     global loslog
+    global cpalog
     conflog = datalog.crelog('USEPECONFLOG', None, confheader)
     loslog = datalog.crelog('USEPELOSLOG', None, losheader)
+    cpalog = datalog.crelog('USEPECPALOG', None, cpaheader)
 
     # Configuration parameters
     config = {
         'plugin_name':     'USEPELOGGER',
         'plugin_type':     'sim',
-        'update_interval': 5.0,
+        'update_interval': 1.0,
```

Comment:
Have you tested the change in accuracy when varying the update interval?

Reply:
There is very little difference in accuracy when varying the interval, especially after the CPA has been passed (less than 1 cm).
However, the estimation of the CPA only works while the conflict is in effect, which ends shortly after CPA has been passed.
Therefore, I wanted to log it as soon as possible by narrowing the interval, as well as getting a more precise timing for the event.